dpgen/generator/run.py

```diff
@@ -1771,6 +1772,41 @@ def make_fp_pwscf(iter_index,
     # link pp files
     _link_fp_vasp_pp(iter_index, jdata)
 
+def make_fp_abacus_pw_scf(iter_index,
+                  jdata) :
+    # make config
+    fp_tasks = _make_fp_vasp_configs(iter_index, jdata)
+    if len(fp_tasks) == 0 :
+        return
+    # make abacus/pw/scf input
+    iter_name = make_iter_name(iter_index)
+    work_path = os.path.join(iter_name, fp_name)
+    fp_pp_files = jdata['fp_pp_files']
+    if 'user_fp_params' in jdata.keys() :
+        fp_params = jdata['user_fp_params']
+        #user_input = True
```

Comment:
Are these codes similar to `make_fp_abacus_pw` in `relabel.py`? Do you consider merging them?

Reply:
I cannot do this. Actually I did not thoroughly read the Run process workflow, so I just wrote the interface imitating the pwscf interface. And I found that there are two sets of functions making input files in run.py and relabel.py for every first principle software, such as `make_fp_siesta` and `make_siesta`, `make_fp_pwscf` and `makepwscf`. I followed these writing mode.

Reply:
@amcadmus Are we stilling use "relabel.py"? Do you think it necessary to add unit tests for this function?

Reply:
> That's my point. "relabel.py" serves to re-calculate the first-principles jobs if there is need. Actually I haven't use this fucntion, either. If you cannot use that function and make sure that your modification works, you should't write codes based on it. Just go ahead.

Let me try the work flow out without the codes in "relabel.py". If it works, I will remove these codes.

Reply:
Yes we are using relabel.py for relabeling a system under different fp settings.